rules/calendar-do-you-allow-full-access-to-calendar-admins/rule.md

```diff
@@ -35,6 +35,8 @@ Always make sure that:
 
   ![Figure: Give admins full access to your calendar in Outlook with "Can edit"](Permissions.jpg)
 
+![Figure: For Sydney Admins look for "SSWSydneyAdmins"](sswsydneyadmin.png)
```

Comment:
hi @LuciaSirtori 

We make Rules generic enough so other people outside SSW can also rely on them. This change makes it very specific to SSW. What's the story behind this change?